docs/application/dotnet/samples/ui/overview.md

```diff
@@ -1,12 +1,13 @@
 # UI
-
+**The meaning of tag**
```

Comment:
It looks like this description for M,W, and T should be added to other sample pages, too.

Actually I think it would be much better to add this describe at cover page only instead if adding to every pages,
but we don't have cover page to write it under Sample folder.

So, would you add this "**The meaning of tag**" to each pages?

Reply:
done